api/controllers/accounts.js

```diff
@@ -21,16 +21,23 @@ var swaggerHelper = require('../../helpers/swagger');
 var modules;
 
 /**
- * Initializes with scope content and private variables:
- * - modules
- * @class AccountsController
- * @classdesc Main System methods.
- * @param {scope} scope - App instance.
+ * @class
+ * @memberof controllers
+ * @requires lodash
```

Comment:
Do we want these `require`s on the class level, or the method level?

Reply:
In the JsDoc- Documentation, it is set at the class level, see: http://usejsdoc.org/tags-requires.html